exdb/models.py

```diff
@@ -99,6 +100,44 @@ class Section(models.Model):
     def __str__(self):
         return self.name
 
+    def completion_board_stuff(self):
```

Comment:
Also a comment describing what is going on and how to use this method would be helpful.